dashboard/app/models/pd/workshop_constants.rb

```diff
@@ -87,5 +87,9 @@ module WorkshopConstants
       COURSE_CSD => {course_name: 'csd-2018'},
       COURSE_CSP => {course_name: 'csp-2018'}
     }.freeze
+
+    # Pilot CSF 201 workshops from 4/24/19 to 5/6/19
+    # that we want to block reminders and exit emails
+    BLOCKED_CSF_201_WORKSHOPS = [6545, 6410, 6543]
```

Comment:
I would prefer this to be captured by adding a `prevent_sending_emails` property to the workshop model - it seems unnecessarily fragile to rely on these ids remaining the same.

Reply:
by 'property' I mean adding a `properties` column for serialized attributes, like we have on applications and a number of other models

Reply:
This could also be a good approach. We end up using time constraint because # of pilot workshop is unknown.